python/dataproc_templates/azure/README.md

```diff
@@ -0,0 +1,80 @@
+# Azure To BigQuery
+
+Template for reading files from Azure Blob Storage and writing them to a BigQuery table. It supports reading JSON, CSV, Parquet, Avro and Delta.IO formats.
+
+It uses the [Spark BigQuery connector](https://cloud.google.com/dataproc-serverless/docs/guides/bigquery-connector-spark-example) for writing to BigQuery.
+
+## Arguments
+
+* `azure.storage.account`: Azure Storage Account  
+* `azure.container.name`:  Azure Container Name
+* `azure.sas`: Azure SAS Token
+* `azure.bigquery.input.location`: Azure blob of the input files (format: `"wasbs://{azure.container.name}@{azure.storage.account}.blob.core.windows.net/{input_location}"`)
+* `azure.bigquery.output.dataset`: BigQuery dataset for the output table
+* `azure.bigquery.output.table`: BigQuery output table name
+* `azure.bigquery.input.format`: Input file format (one of: avro,parquet,csv,json)
+* `azure.bigquery.temp.bucket.name`: Temporary bucket for the Spark BigQuery connector
+* `azure.bigquery.output.mode`: Output write mode (one of: append,overwrite,ignore,errorifexists)(Defaults to append)
```

Comment:
Bit unconvinced from the nomenclature we are following here. Using **Azure** alone in the template name and the template properties doesn't make sense as we are exporting data specifically from **Azure Blob Storage**.

Reply:
i believe azure convention is Azure Storage Account not Azure Blob Storage Account and the same for Azure Container Name. should we rename them?

Reply:
So, Using Azure Storage Account also would not be correct here. As a Storage Account can support services for Blob, Tables, Queues, ADLS etc. 
Since, we are dealing with only blob storage here, we should be good with using _Azure Blob Storage_ in the template name , and _azure.blob.bigquery*_ in the templateProperty.